arm-storage/2016-05-01/swagger/storage.json

```diff
@@ -0,0 +1,1301 @@
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used for viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key with a length no greater than 128 characters and a value with a length no greater than 256 characters."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/StorageAccountPropertiesCreateParameters"
+        }
+      },
+      "required": [
+        "sku",
+        "kind",
+        "location"
+      ],
+      "description": "The parameters used when creating a storage account."
+    },
+    "Endpoints": {
+      "properties": {
+        "blob": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the blob endpoint."
+        },
+        "queue": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the queue endpoint."
+        },
+        "table": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the table endpoint."
+        },
+        "file": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the file endpoint."
+        }
+      },
+      "description": "The URIs that are used to perform a retrieval of a public blob, queue, or table object."
+    },
+    "StorageAccountProperties": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the status of the storage account at the time the operation was called.",
+          "enum": [
+            "Creating",
+            "ResolvingDNS",
+            "Succeeded"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": false
+          }
+        },
+        "primaryEndpoints": {
+          "$ref": "#/definitions/Endpoints",
+          "readOnly": true,
+          "description": "Gets the URLs that are used to perform a retrieval of a public blob, queue, or table object. Note that Standard_ZRS and Premium_LRS accounts only return the blob endpoint."
+        },
+        "primaryLocation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the location of the primary data center for the storage account."
+        },
+        "statusOfPrimary": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the status indicating whether the primary location of the storage account is available or unavailable.",
+          "enum": [
+            "Available",
+            "Unavailable"
+          ],
+          "x-ms-enum": {
+            "name": "AccountStatus",
+            "modelAsString": false
+          }
```

Comment:
[This is the format](https://github.com/Azure/openapi-validation-tools/blob/master/arm-storage/2016-01-01/swagger/storage.json#L65) in which we are providing samples for an operation. 
[This is the actual sample](https://github.com/Azure/openapi-validation-tools/blob/master/arm-storage/2016-01-01/examples/storageAccountCreate.json) in a separate json file.
It captures example/sample values for everything that the user needs to make a request (path/query/body parameters). It also has a sample value for the response body and any important headers (for long running operations). This helps us ensure that the actual wire format and the swagger spec are in sync. The tool validates all the sample values against the spec.

Moving forward we would want all the service teams to provide samples as a part of their PR. It would be great if you can use the sample that we already have for the 2016-01-01 api-version of storage spec and add samples for the 2 new operations added in this api-version. 

the doc team is going to take all these samples and lighten them in the docs.microsoft.com website in the next sprint.